subsys/fs/pm.yml

```diff
@@ -0,0 +1,7 @@
+#include <autoconf.h>
+
+#ifdef CONFIG_FILE_SYSTEM_NFFS
```

Comment:
Is this needed? I would guess it's covered by the corresponding `if` in the CmakeLists.txt.

Reply:
true, but it shows how to handle multiple subsystems defined in `fs`. That way each of them can configure in their own partition.

Reply:
In that case, should the other be ` if (CONFIG_FILE_SYSTEM_NFFS)` or something?

Reply:
Not sure I get what you mean.  

One example could be
```
#ifdef CONFIG_FILE_SYSTEM_NFFS
nffs_storage:
  blabalbla
#endif

#ifdef CONFIG_FILE_SYSTEM_LITTLEFS
littlefs_storage:
  something: blabal
#endif
```

Does that answer your question?

Reply:
Sorry, I meant `if (CONFIG_FILE_SYSTEM_FS)`

Reply:
Correct